govc/vm/info.go

```diff
@@ -101,12 +101,10 @@ func (cmd *info) Run(ctx context.Context, f *flag.FlagSet) error {
 	}
 
 	vms, err := cmd.VirtualMachines(f.Args())
-	if err != nil {
-		if _, ok := err.(*find.NotFoundError); ok {
-			// Continue with empty VM slice
-		} else {
-			return err
-		}
+	if _, ok := err.(*find.NotFoundError); ok {
+		// Continue with empty VM slice
+	} else {
+		return err
```

Comment:
We don't want to return `err` here if it is `nil`, looks like this is what's causing CI to fail - at least if fails locally for me unless this hunk is reverted.

Reply:
@dougm Good catch! I reverted it for the moment. Thanks!